tools/azure-sdk-tools/devtools_testutils/keyvault_testcase.py

```diff
@@ -0,0 +1,129 @@
+from collections import namedtuple
+import os
+import time
+
+from azure.mgmt.keyvault import KeyVaultManagementClient
+from azure.mgmt.keyvault.models import (
+    SecretPermissions,
+    KeyPermissions,
+    CertificatePermissions,
+    StoragePermissions,
+    Permissions,
+    Sku,
+    SkuName,
+    AccessPolicyEntry,
+    VaultProperties,
+    VaultCreateOrUpdateParameters,
+)
+
+from azure_devtools.scenario_tests.exceptions import AzureTestError
+
+from . import AzureMgmtPreparer, ResourceGroupPreparer, FakeResource
+from .resource_testcase import RESOURCE_GROUP_PARAM
+
+
+DEFAULT_PERMISSIONS = Permissions(
+    keys=[perm.value for perm in KeyPermissions],
+    secrets=[perm.value for perm in SecretPermissions],
+    certificates=[perm.value for perm in CertificatePermissions],
+    storage=[perm.value for perm in StoragePermissions],
+)
+DEFAULT_SKU = SkuName.premium.value
+CLIENT_OID = "00000000-0000-0000-0000-000000000000"
+
+class KeyVaultAccountPreparer(AzureMgmtPreparer):
```

Comment:
KeyVault~~Account~~Preparer? I know what a "key vault" is but not what a "key vault account" would be. Also, module name is `keyvault_testcase` but there's no `TestCase` here.

Reply:
switched to KeyVaultPreparer